extras/docker/rpi/heketi-start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -f /backupdb/heketi.db ] ; then
-    cp /backupdb/heketi.db /var/lib/heketi/heketi.db
+if [ -f /backupdb/heketi.db.gz ] ; then
+    gunzip -c /backupdb/heketi.db.gz > /var/lib/heketi/heketi.db
```

Comment:
Ah, ok, it then should handle either heketi.db _or_ heketi.db.gz. I think that would fix it.  What do you guys think? @obnoxxx @raghavendra-talur